assets/projects.json

```diff
@@ -160,5 +160,27 @@
       "medium"
     ],
     "description": "Design an api service which can be used to easily exchange small code snippets between developers. These snippets should be search-able on given params. In addition, if a search is done on multiple codesnippets, return this result with pagination."
+  },
+  {
+    "title": "Math game",
+    "tags": [
+      "easy",
+      "javascript",
+      "react native",
+      "typescript",
+      "futter",
```

Comment:
Don't list languages and frameworks, you could technically create this using anything.